README.md

```diff
@@ -1,68 +1,106 @@
+Resolver.sonatypeOssRepos("snapshots") // for snaphots
 // `<latest version>` refers to the version indicated by the badge above
 libraryDependencies ++= Seq(
-  "io.pdal" %% "pdal" % "<latest version>", // core library
-  "io.pdal" %  "pdal-native" % "<latest version>" // jni bindings
+"io.pdal" %% "pdal" % "<latest version>", // core library
+"io.pdal" %  "pdal-native" % "<latest version>" // jni bindings
 )
 ```
-
 If you would like to use your own bindings, it is necessary to set `java.library.path`:
-
 ```scala
 // Mac OS X example with manual JNI installation
 // cp -f native/target/resource_managed/main/native/x86_64-darwin/libpdaljni.2.1.dylib /usr/local/lib/libpdaljni.2.1.dylib
 // place built binary into /usr/local/lib, and pass java.library.path to your JVM
 javaOptions += "-Djava.library.path=/usr/local/lib"
 ```
 
-You can use `pdal-native` dep in case you don't have installed JNI bindings and to avoid steps described above.
-Dependency contains bindings for `x86_64-darwin` and `x86_64-linux`, other versions are not supported yet.
-
-## PDAL-Scala (Scala 2.x)
 
+### Pdal-Scala (Scala 2.X)
 Scala API allows to build pipeline expressions instead of writing a raw JSON.
-
 ```scala
 // `<latest version>` refers to the version indicated by the badge above
 libraryDependencies ++= Seq(
-  "io.pdal" %% "pdal-scala" % "<latest version>", // scala core library
-  "io.pdal" %  "pdal-native" % "<latest version>" // jni bindings
+"io.pdal" %% "pdal-scala" % "<latest version>", // scala core library
+"io.pdal" %  "pdal-native" % "<latest version>" // jni bindings
 )
 ```
-
-Scala API covers PDAL 2.0.x, to use any custom DSL that is not covered by the 
+Scala API covers PDAL 2.0.x, to use any custom DSL that is not covered by the
 current Scala API you can use `RawExpr` type to build `Pipeline Expression`.
 
-## Code examples
+## Build
+Development purposes (including binaries) compilation:
+1. Install PDAL (using brew / package managers (unix) / build from sources / etc)
+2. Install sbt (using brew / package managers (unix))
+3. Build native libs `sbt native/nativeCompile` (optionally, binaries would be built during tests run)
+4. Run `sbt core/test` to run PDAL tests
+Only Java development purposes compilation:
+1. Provide `$LD_LIBRARY_PATH` or `$DYLD_LIBRARY_PATH`
+2. If you don't want to provide global variable you can pass `-Djava.library.path=<path>` into sbt:
+`./sbt -Djava.library.path=<path>`
+3. Set `PDAL_DEPEND_ON_NATIVE=false` (to disable `native` project build)
+4. Run `PDAL_DEPEND_ON_NATIVE=false sbt`
+Finally the possible command to launch and build PDAL JNI bindings could be:
+```bash
+# Including binaries build
+sbt
+```
+```bash
+# Java side development without binaries build
+PDAL_DEPEND_ON_NATIVE=false sbt -Djava.library.path=<path>
+```
 
-#### Demo project with examples
 
-JNI bindings basic usage examples can be found [here](./examples).
 
-### PDAL Core (Scala 2.x / 3.x)
 
+### Mac-OS ARM tutorial
```

Comment:
ARM tutorial is really no different than the build instructions above. It's also not a "tutorial"?

I'd replace this section by mentioning somewhere explicitly that arm binaries are not prebuilt and it requires manual compilation.